microbench/src/main/java/io/netty5/microbench/handler/ssl/AbstractSslEngineBenchmark.java

```diff
@@ -15,7 +15,7 @@
  */
 package io.netty5.microbench.handler.ssl;
 
-import io.netty.buffer.ByteBufAllocator;
+import io.netty5.buffer.api.BufferAllocator;
```

Comment:
This would be actually interesting... Can you run the benchmark for 4.1 and compare to this branch ?

Reply:
Performance is currently half of 4.1, because the `PooledBufferAllocator` has gotten slower, it seems.